spec/random.js

```diff
@@ -0,0 +1,6 @@
+// module.exports = function(myArray) {
```

Comment:
are you still going to work on this function? if not I would just remove the file from you repo